modules/apps/site/site-api/src/main/java/com/liferay/site/display/context/GroupDisplayContextHelper.java

```diff
@@ -25,6 +31,10 @@ public GroupDisplayContextHelper(HttpServletRequest httpServletRequest) {
 		_httpServletRequest = httpServletRequest;
 	}
 
+	public List<BreadcrumbEntry> getBreadcrumbEntries() {
+		return ListUtil.fromArray(_getStagingBreadcrumbEntry());
+	}
+
```

Comment:
This will create an array just to convert it into a list, which is unnecessary. In cases like this use `Collections.singletonList()`, and in general, avoid creating unnecessary intermediate results.